us/en/skymiles/blocks/blade/blade.js

```diff
@@ -6,13 +6,14 @@ export default function decorate(block) {
       } else {
         div.className = 'blade-body';
         const a = div.lastElementChild.querySelector('a:only-child:last-child');
+        a.classList.add('wrap');
```

Comment:
If you end up not using the `.wrap` selector, we can probably also just remove the class decoration